src/main/kotlin/org/malachite/estella/admin/api/AdminController.kt

```diff
@@ -34,25 +34,17 @@ class AdminController(
             UNAUTH
 
 
-
     @PostMapping("/deverify/{organizationUUID}")
     fun deverifyOrganization(
         @RequestHeader(EStellaHeaders.adminApiKey) apiKey: String?,
         @PathVariable organizationUUID: String): ResponseEntity<Any> =
         if(securityService.isCorrectApiKey(apiKey))
             organizationService
                 .deverifyOrganization(organizationUUID)
-                .let { SUCCESS}
+                .let { SUCCESS }
         else
             UNAUTH
 
-
-    fun Organization.toDto() = OrganizationDto(
```

Comment:
Nigdzie nie będziemy potrzebować OrganizationDto?

Reply:
Ono jest raczej przeniesione, a nie wyrzucone, z tego, co pamiętam, a jeśli nie, to pewnie gdzieś jest analogiczne, albo w ogóle nie było używane